dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionRestRepositoryIT.java

```diff
@@ -3200,4 +3205,342 @@ public void addColAdminGroupToCheckReindexingTest() throws Exception {
                                .andExpect(jsonPath("$.page.totalElements", is(1)));
     }
 
+    @Test
```

Comment:
Is there any test that verifies the new methods only return collection the user can submit to (verifies the submit rights)